src/components/WelcomeModal/CarouselDots.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { CarouselRef } from "antd/es/carousel";
+import { size } from "constants/tokens";
+
+const { gray, green } = uiColors;
+
+interface CarouselDotProps {
+  activeSlide: number;
+  total: number;
+  slider: React.MutableRefObject<CarouselRef>;
+}
+
+const CarouselDots: React.VFC<CarouselDotProps> = ({
+  activeSlide,
+  total,
+  slider,
+}) => (
+  <DotContainer>
+    {Array.from({ length: total }).map((val, index) => (
```

Comment:
Something strange I'm seeing is that the active dot won't update for me for the first click 🤔 But it works every time after

https://user-images.githubusercontent.com/47064971/167454629-1c81c457-a296-4b04-9000-00bb3d38e7a1.mov

Reply:
lmk if you notice it again i reworked some state logic.